_pages/install/integrations/opf-integration/opf-child-pages/3-order-confirmation-scripts.md

```diff
@@ -0,0 +1,48 @@
+
+## Enabling Order Confirmation Scripts
+
+You can enable order confirmation scripts by installing the `@spartacus/opf` OPF library. For more information, see [Installing Additional Spartacus Libraries]({{ site.baseurl }}/schematics/#installing-additional-spartacus-libraries).
+
+### CMS Components
+
+Order confirmation scripts are CMS-driven and consist of the following CMS component:
+
+-`OpfCtaScriptsComponent`
+
+Order confirmation scripts and on-site messaging both use the CTA scripts CMS component, `OpfCtaScriptsComponent`. The order confirmation scripts feature is enabled when the `OpfCtaScriptsComponent` CMS component is enabled on the Order Confirmation or Order Details pages. The on-site messaging feature is enabled when the `OpfCtaScriptsComponent` CMS component is enabled on the Product Details or Cart pages.
+
+You can configure order confirmation scripts by using SmartEdit to display the CTA scripts component in Spartacus, or you can manually add it to content slots using ImpEx.
+
+If you are using the [{% assign linkedpage = site.pages | where: "name", "spartacussampledata-extension.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/install/spartacussampledata-extension.md %}), the CTA scripts component is already enabled. However, if you decide not to use the `spartacussampledata` extension, you can enable the CTA scripts component through ImpEx.
+
+**Note:** The `$contentCV` variable, which stores information about the content catalog, and which is used throughout the ImpEx in the following procedures, is defined as follows:
+
+```text
+$contentCatalog=electronics-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Online])[default=$contentCatalog:Online]
+```
+
+### Adding CMS Components Manually
+
+This section describes how to add the CTA scripts CMS component to Spartacus using ImpEx.
+
+Add the CTA scripts CMS component to the Order Confirmation and Order Details pages with the following ImpEx:
+
+```text
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType
+;;OpfCtaScriptsComponent;Opf Cta Scripts Component;OpfCtaScriptsComponent
+
+UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;active;cmsComponents(uid,$contentCV)
+;;ProductSummarySlot;Site Context Slot;true;ProductImagesComponent, ProductIntroComponent, QualtricsEmbeddedFeedbackComponent, ProductSummaryComponent, VariantSelector, ConfigureProductComponent, AddToWishListComponent, StockNotificationComponent, OpfCtaScriptsComponent, AddToCart
+;;TopContent-cartPage;Top content for Cart Slot;true;OpfCtaScriptsComponent, AddToSavedCartsComponent, CartComponent, ClearCartComponent, SaveForLaterComponent, ImportExportOrderEntriesComponent
+```
+
```

Comment:
@FollowTheFlo @Matejk00 Can you double-check this ImpEx? The text says it adds CTA scripts to the Order Confirmation and Order Details pages, but it looks like it is adding only to the Cart Page (probably this ImpEx was intended for on-site mesage?)

Reply:
It looks like an impex for the on-site messaging. 
For CTA we should use the following impex:

```
INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType
;;OpfCtaScriptsComponent;Opf Cta Scripts Component;OpfCtaScriptsComponent

# Add CTA script order confirmation content slot
UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;active;cmsComponents(uid,$contentCV)
;;BodyContent-orderConfirmation;Body Content Slot for Order Confirmation;true;OpfCtaScriptsComponent, OrderConfirmationThankMessageComponent, OrderConfirmationShippingComponent, OrderConfirmationPickUpComponent, ExportOrderEntriesComponent, OrderConfirmationBillingComponent, OrderConfirmationTotalsComponent, OrderConfirmationContinueButtonComponent

# Add CTA script to OPF order details page content slot
UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;active;cmsComponents(uid,$contentCV)
;;BodyContent-orderdetail;Body Content Slot for My Account Order Details;true;OpfCtaScriptsComponent,AccountOrderDetailsSimpleOverviewComponent,AccountOrderDetailsGroupedItemsComponent,ExportOrderEntriesComponent,AccountOrderDetailsBillingComponent,AccountOrderDetailsTotalsComponent,AccountOrderDetailsActionsComponent
```

Reply:
Awesome!! Thank you!! 🙏 🙂